ArmPkg/Include/Library/TpmServiceLib.h

```diff
@@ -0,0 +1,77 @@
+/** @file
+  Definitions for the TPM Service
+
+  Copyright (c), Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef TPM_SERVICE_LIB_H_
+#define TPM_SERVICE_LIB_H_
+
+#include <Base.h>
+#include <IndustryStandard/ArmFfaPartInfo.h>
+#include <Library/ArmSvcLib.h>
+#include <Library/ArmFfaLibEx.h>
+
+typedef INT64 TpmStatus;
+
+// Status Codes spec defined in: https://developer.arm.com/documentation/den0138/0100/?lang=en
+#define TPM_STATUS_OK          ((TpmStatus)0x05000001)
+#define TPM_STATUS_OK_RESULTS  ((TpmStatus)0x05000002)
+// Error Status
+#define TPM_STATUS_NO_FUNC            ((TpmStatus)0x8e000001)
+#define TPM_STATUS_NOT_SUP            ((TpmStatus)0x8e000002)
+#define TPM_STATUS_INVARG             ((TpmStatus)0x8e000005)
+#define TPM_STATUS_INV_CRB_CTRL_DATA  ((TpmStatus)0x8e000006)
+#define TPM_STATUS_ALREADY            ((TpmStatus)0x8e000009)
+#define TPM_STATUS_DENIED             ((TpmStatus)0x8e00000A)
+#define TPM_STATUS_NOMEM              ((TpmStatus)0x8e00000B)
+
+/* Opcodes
+ * TPM Opcodes spec defined in: https://developer.arm.com/documentation/den0138/0100/?lang=en */
+#define TPM_OPCODE_BASE                   (0x0F000000)
+#define TPM_OPCODE_GET_INTERFACE_VERSION  (TPM_OPCODE_BASE + 0x01)
+#define TPM_OPCODE_GET_FEATURE_VERSION    (TPM_OPCODE_BASE + 0x101)

```

Comment:
This is already defined in the `Guid/Tpm2ServiceFfa.h`. Probably should reuse those?